src/CategoricalArrays.jl

```diff
@@ -1,13 +1,14 @@
 __precompile__()
 module CategoricalArrays
-    export CategoricalPool, CategoricalValue
+    export CategoricalPool, CategoricalValue, CategoricalString
     export AbstractCategoricalArray, AbstractCategoricalVector, AbstractCategoricalMatrix,
            CategoricalArray, CategoricalVector, CategoricalMatrix
     export AbstractNullableCategoricalArray, AbstractNullableCategoricalVector,
            AbstractNullableCategoricalMatrix,
            NullableCategoricalArray, NullableCategoricalVector, NullableCategoricalMatrix
     export LevelsException
 
+    export catvalue#, iscatvalue
```

Comment:
?

Reply:
`catvalue` is the function that takes the place of `CategoricalValue` *function*
For `iscatvalue()` you can decide whether to export it or not or keep it as a remainder to export in the future.

Reply:
Better not export any of these (and not leave a comment). Anyway `catvalue` only works if you access the pool, which isn't exported currently either.

Reply:
OK, `CategoricalValue()` method "leaked" with the `CategoricalValue` type and was used in the tests, so I'll remove the exports and fix the tests to use `CategoricalArrays.catvalue()`.